compiler/webpack-loader/lib/loader.js

```diff
@@ -0,0 +1,106 @@
+const path = require('path')
+const fs = require('fs')
+const qs = require('querystring')
+const { getOptions } = require('loader-utils')
+const { exec } = require('child_process')
+
+const IMPORT_REGEX = /import [a-zA-Z]+ from "([a-zA-Z./]+)"/g
```

Comment:
Depending on the length of the imports, there might be newlines between the different parts of this, so we might want to match arbitrary whitespace.

In some rare cases I think we might actually import things other than the default component, e.g. `import Button, { BUTTON_TYPES } from...`. But we can handle that later after we graduate from regex.

Reply:
Yeah this is just a hack until we get dependencies info from the lona compiler directly